node/src/manager/commands/index.rs

```diff
@@ -41,3 +41,34 @@ pub async fn create(
         Err(other) => Err(anyhow::anyhow!(other)),
     }
 }
+
+pub async fn list(
+    store: Arc<SubgraphStore>,
+    id: String,
+    entity_name: String,
+) -> Result<(), anyhow::Error> {
+    let deployment_hash = DeploymentHash::new(id)
```

Comment:
It would be desirable to take a namespace `sgd1234` in all places where a deployment hash is allowed. A lot of times, those are easier to work with than deployment hashes.